tools/BaseImageUpdate/baseImage.ps1

```diff
@@ -40,17 +48,28 @@ function Update-ARM-BaseImages
         #>
         [Parameter(Mandatory)]
         [string]
-        $NewASPNetCoreVersion
-    )
+        $NewASPNetCoreVersion,
 
-    Setup-BaseImage-Script
+        <# 
+        Object array contain file paths to base image dockerfile
+        #>
+        [Parameter(Mandatory=$false)]
+        [Object[]]
+        $FileLocale
```

Comment:
In what case is this param needed?

Reply:
Good question. I was thinking that I should have a function that calls both `Update-ARM-BaseImages` and `Update-AMD64-BaseImages`. This param will allow traversing the folder structure twice. I suppose I can create that function now both of the sub functions are ready.